diff --git a/node_modules/@types/three/src/animation/tracks/QuaternionKeyframeTrack.d.ts b/node_modules/@types/three/src/animation/tracks/QuaternionKeyframeTrack.d.ts
index 4bcaa98..f1b9eac 100644
--- a/node_modules/@types/three/src/animation/tracks/QuaternionKeyframeTrack.d.ts
+++ b/node_modules/@types/three/src/animation/tracks/QuaternionKeyframeTrack.d.ts
@@ -9,8 +9,8 @@ export class QuaternionKeyframeTrack extends KeyframeTrack {
      * Constructs a new Quaternion keyframe track.
      *
      * @param {string} name - The keyframe track's name.
-     * @param {Array<number>} times - A list of keyframe times.
-     * @param {Array<number>} values - A list of keyframe values.
+     * @param {ArrayLike<number>} times - A list of keyframe times.
+     * @param {ArrayLike<number>} values - A list of keyframe values.
      * @param {(InterpolateLinear|InterpolateDiscrete|InterpolateSmooth)} [interpolation] - The interpolation type.
      */
     constructor(name: string, times: ArrayLike<number>, values: ArrayLike<number>, interpolation?: InterpolationModes);
diff --git a/node_modules/@types/three/src/nodes/core/LightingModel.d.ts b/node_modules/@types/three/src/nodes/core/LightingModel.d.ts
index cb894c0..fbbc46f 100644
--- a/node_modules/@types/three/src/nodes/core/LightingModel.d.ts
+++ b/node_modules/@types/three/src/nodes/core/LightingModel.d.ts
@@ -10,6 +10,8 @@ export interface LightingModelReflectedLight {
 }
 
 export interface LightingModelDirectInput extends DirectLightData {
+    lightColor: Node;
+    lightDirection: Node;
     lightNode: Node;
     reflectedLight: LightingModelReflectedLight;
 }
diff --git a/node_modules/@types/three/src/nodes/core/NodeBuilder.d.ts b/node_modules/@types/three/src/nodes/core/NodeBuilder.d.ts
index eb477e3..0be5b27 100644
--- a/node_modules/@types/three/src/nodes/core/NodeBuilder.d.ts
+++ b/node_modules/@types/three/src/nodes/core/NodeBuilder.d.ts
@@ -49,6 +49,8 @@ export default abstract class NodeBuilder {
     cache: NodeCache;
     globalCache: NodeCache;
 
+    context: NodeBuilderContext;
+
     /**
      * @TODO used to be missing. check the actual type later
      */
