diff --git a/node_modules/@types/three/src/core/BufferGeometry.d.ts b/node_modules/@types/three/src/core/BufferGeometry.d.ts
index 0f129f1..153a564 100644
--- a/node_modules/@types/three/src/core/BufferGeometry.d.ts
+++ b/node_modules/@types/three/src/core/BufferGeometry.d.ts
@@ -172,7 +172,7 @@ export class BufferGeometry<
      * You will have to call {@link dispose | .dispose}(), and create a new instance of {@link THREE.BufferGeometry | BufferGeometry}.
      * @defaultValue `{}`
      */
-    morphAttributes: Record<"position" | "normal" | "color", Array<BufferAttribute | InterleavedBufferAttribute>>;
+    morphAttributes: Partial<Record<"position" | "normal" | "color", Array<BufferAttribute | InterleavedBufferAttribute>>>;
 
     /**
      * Used to control the morph target behavior; when set to true, the morph target data is treated as relative offsets, rather than as absolute positions/normals.
